src/map/zone_entities.cpp

```diff
@@ -485,6 +485,22 @@ void CZoneEntities::DecreaseZoneCounter(CCharEntity* PChar)
         }
     }
 
+    // Duplicated from charUtils, it is theoretically possible through d/c magic to hit this block and not sendToZone
```

Comment:
Just to be anal, can this be extracted out into something in charutils or synthutils? `failSynthOnZone` or something

Reply:
updated